WolvenKit/Views/Documents/RDTTextureView.xaml.cs

```diff
@@ -222,6 +246,11 @@ public void ResetZoomPan(object sender, RoutedEventArgs e)
             pan.Y = 0;
             end.X = 0;
             end.Y = 0;
+            var temp_filename = Path.Combine(ISettingsManager.GetTemp_OBJPath(), "embedded_preview.png");
```

Comment:
What is this?

Reply:
something I should have deleted. I made the reset zoom menu item do the save first then added the other one, forgot to delete that code.